packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -20,23 +20,42 @@ import { IRouterliciousDriverPolicies } from "./policies";
 import { ICache } from "./cache";
 import { ISnapshotTreeVersion } from "./definitions";
 
+/**
+ * Amount of time after disconnect within which we don't need to rediscover.
+ * In the future, we likely want to retrieve this information from service's "inactive session" definition.
+ */
+const RediscoverAfterTimeDisconnectedMs = 60000; // 1 minute
+
 /**
  * The DocumentService manages the Socket.IO connection and manages routing requests to connected
  * clients
  */
 export class DocumentService implements api.IDocumentService {
+    private lastDisconnectAt: number = Date.now();
```

Comment:
If this is initialized to now(), it will probably not be 60s before a method like connectToDeltaStorage() is called for the first time, right? And don't we want that to go through discovery?

Reply:
Discovery is called immediately before the DocumentService is created the first time (in DocumentServiceFactory) so we can safely assume that discovery will have happened within 60s (likely <1ms) of this now() timestamp. We do not want to duplicate the first discovery call that is already being made.